code/notificationbuilder/src/main/java/com/adobe/marketing/mobile/notificationbuilder/NotificationBuilder.kt

```diff
@@ -0,0 +1,24 @@
+/*
+  Copyright 2024 Adobe. All rights reserved.
+  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License. You may obtain a copy
+  of the License at http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software distributed under
+  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+  OF ANY KIND, either express or implied. See the License for the specific language
+  governing permissions and limitations under the License.
+*/
+
+package com.adobe.marketing.mobile.notificationbuilder;
+
+import kotlin.jvm.JvmStatic;
+
+object NotificationBuilder {
+    private const val SELF_TAG = "NotificationBuilder"
+    const val VERSION = "3.0.0"
+
+    @JvmStatic
+    fun extensionVersion(): String {
```

Comment:
`NotificationBuilder` is not an extension. Can you rename this function to `version()` ? 
@prudrabhat Thoughts?

Reply:
Good point... I am voting for just version() as well

Reply:
Yea. I am aligned on `version()`  👍🏼